### DISCUSSION TITLE
Searching for "Display beatmaps from" in settings should also show the "up to" control

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#18465] [Ideas] [answered]
If you search for the "Display beatmaps from" slider in the settings field, you can't find the corresponding "up to" slider. This is somewhat unintuitive, in my opinion.

Making it so you find both while searching for either, would be a nice change?

Version: 2022.528.0

**Comment by bdach** [comment]
Probably not worth the hassle. That feature is a source of frequent confusion and will more likely than not be straight up removed.

**Reply by peppy** [reply]
And before anyone complains that they use this and don't want to see it removed: it will definitely have an alternative available in song select, rather than at the weird location in global settings.

**Comment by Gabixel** [comment, marked as answer]
The filter has been removed from the settings and is now visible in the carousel as a range slider (#18962)
